tests/suites/network/network_health.sh

```diff
@@ -8,7 +8,6 @@ run_network_health() {
     # Deploy some applications for different series.
     juju deploy mongodb --series xenial
     juju deploy ubuntu ubuntu-bionic --series bionic
-    juju deploy ubuntu ubuntu-focal --series focal
```

Comment:
Have you opened a bug to track updating the charm to support focal? I think this is fairly important as focal is our default series now.